### PR TITLE
Allow InboxRecyclerView to work in the design view

### PR DESCRIPTION
**Pull request by chrisbanes**
The fake adapter Android Studio uses does not support stable IDs, so we need to relax the requirement in edit mode.

### CODE REVIEW

**Review by saket — APPROVED** [review]

**Comment by saket** [comment]
🙌

This removes the preview error, but the layout preview still doesn't work with `tools:listitem`. Would you know why? It seems like the preview renderer doesn't like subclasses of `RecyclerView`.

I created a sample project to reproduce this in isolation: 
https://github.com/saket/RecyclerViewPreviewBug

**Comment by chrisbanes** [comment]
FYI, I can't merge this change since I don't have write access 😄 

> This removes the preview error, but the layout preview still doesn't work with tools:listitem. Would you know why? It seems like the preview renderer doesn't like subclasses of RecyclerView.

Not sure, sounds like a bug in the layout editor.

**Comment by saket** [comment]
Right, I just wanted to confirm this in case more changes were needed to support layout preview. I will go ahead raise a bug report. Thanks for sending this PR!

**Comment by saket** [comment]
Pushed a snapshot build to maven with this change
```
me.saket:inboxrecyclerview:2.0.0-SNAPSHOT
```

**Pull request merged**
